.github/workflows/plugins-ci.yml

```diff
@@ -8,6 +8,11 @@ on:
         required: false
         default: 'fastify'
         type: string
+      database:
+        description: 'Possible values: none, postgres, mongo, mysql, redis'
+        required: false
+        default: none
```

Comment:
I actually wonder if default results in prefilling it or failing then test job, as it could be empty